frontend/public/locales/en/discovery.json

```diff
@@ -3,9 +3,9 @@
     "discovery.edit": "Edit cluster discovery",
     "discovery.import": "Import cluster",
 
-    "disable.title": "Disable cluster discovery",
-    "disable.button": "Disable",
-    "disable.message": "You are disabling all cluster discovery. All connections will be removed from the filter, and the filter and current discovered clusters will be deleted.",
+    "disable.title": "Delete discovery config",
```

Comment:
In these, should "discovery config" be "discoveryConfig" and if not, consider a full phrase: "discovery configuration" please?